setup.py

```diff
@@ -137,6 +137,9 @@ def run_tests(self):
         'python_jsonschema_objects <= 0.3.12',
         'jsonref',
         'markdown<=3.2.2',
+        # Keep importlib-metadata it low, otherwise python_jsonschema_objects
+        # build on python 3.7.12 will not work on maya 2022
```

Comment:
no need to be so specific for the application or python version involved.
first line is enough .